src/data-fetcher/json/json-datafetcher.ts

```diff
@@ -55,28 +57,32 @@ function RawDataFetcher(HGC: any, ...args: any): any {
                 chromLengths: chromosomeSizes
             };
 
+            const { chromosomeField, genomicFields } = this.dataConfig;
+
             this.values = dataConfig.values.map((row: any) => {
                 let successfullyGotChrInfo = true;
 
-                this.dataConfig.genomicFields.forEach((g: any) => {
-                    if (!row[this.dataConfig.chromosomeField]) {
-                        // TODO:
-                        return;
-                    }
-                    try {
-                        const chr = row[this.dataConfig.chromosomeField].includes('chr')
-                            ? row[this.dataConfig.chromosomeField]
-                            : `chr${row[this.dataConfig.chromosomeField]}`;
-                        row[g] = GET_CHROM_SIZES(this.assembly).interval[chr][0] + +row[g];
-                    } catch (e) {
-                        // genomic position did not parse properly
-                        successfullyGotChrInfo = false;
-                        // console.warn(
-                        //     '[Gosling Data Fetcher] Genomic position cannot be parsed correctly.',
-                        //     this.dataConfig.chromosomeField
-                        // );
-                    }
-                });
+                if (genomicFields && chromosomeField) {
```

Comment:
These properties can be undefined, but since we did not use the explicit types, we were ignoring this case.